### ISSUE TITLE
Feature request: print position of mouse in UI

### ISSUE THREAD

**Issue by SpiritSuperUser** [spirit-code/spirit#448]
It would be nice to have the current position of the mouse printed somewhere in the UI. Ie., in "System" visualization the coordinates could be shown while in "Sphere" visualization the corresponding angles are printed.

**Comment by GPMueller** [comment]
Though I agree it would be somewhat nice, would this have any practical use?

**Comment by SpiritSuperUser** [comment]
yes :) if you have a somehow meaningful system and want to place for example a skyrmion at a particular position without the fear of destroying everything by placing it at a wrong position, it would be good to know where. so you check the position of interest with your mouse and then you place the skyrmion there. on the sphere, it would just be nice to have direct access to the orientation of single spins, so you place your mouse on one spin and know theta and phi..

**Comment by GPMueller** [comment]
The mouse position can currently not be calculated by ray-intersection with the bounding box. Therefore, to get reliable numbers, the camera needs to be aligned to the z-axis and use orthogonal projection. Therefore, I would only add this for the interaction modes, in which an additional info-widget could be shown:
<img width="1369" alt="image" src="https://user-images.githubusercontent.com/20167491/104851921-1c9b3700-58f8-11eb-859f-d147b9f4af48.png">

This has been done with 3c7f550d69b72c5c7805a42cb35568d9b0f96af0 on branch `feature-imgui`.
